### DISCUSSION TITLE
[Feature request] Table panel footer: different calculation for different columns

### DISCUSSION THREAD

**Discussion by GedPL** [grafana/grafana#58297] [Dashboards, Panels & Visualizations]
Hello,
When using Table Panel it's possible to enable showing footer and select type of calculation (avg, max, total, etc.) and which columns need to be calculated.
Unfortunately you can select one calculation for all selected columns.
Please consider adding Override allowing to assign different calculations for overrided columns. 

Thanks a lot !
Ged

**Comment by borisdigital** [comment]
Indeed, this would be a really, really nice improvement 👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
